ts/Data/DataTable.ts

```diff
@@ -728,7 +748,13 @@ class DataTable extends DataTableCore implements DataEvent.Emitter {
             column = tableColumns[columnName];
 
             if (column) {
-                columns[columnName] = (asReference ? column : column.slice());
+                if (asReference) {
+                    columns[columnName] = column;
+                } else if (asBasicColumns) {
+                    columns[columnName] = Array.from(column);
+                } else {
```

Comment:
This needs an extra check for performance reasons as a basic column is faster sliced.

Ref: https://www.measurethat.net/Benchmarks/Show/30839/0/slice-vs-array-from#latest_results_block

Reply:
From what I can see, for a typed array `Array.from(typed)` works a bit faster than `[].slice.call(typed)`: [benchmark](https://www.measurethat.net/Benchmarks/Show/33528/0/array-from-vs-slice). I added a condition to the if to execute it only if it's a typed array, otherwise, do `slice`.

Reply:
In Safari and Firefox basic slice is twice and more faster, while Chrome has a slight advantage for Array.from. This needs improvements.